modules/lunamediaBidAdapter.js

```diff
@@ -210,8 +211,18 @@ function createVideoRequestData(bid, bidderRequest) {
   let topLocation = getTopWindowLocation(bidderRequest);
   let topReferrer = getTopWindowReferrer();
 
-  let sizes = getVideoSizes(bid);
-  let firstSize = getFirstSize(sizes);
+  // if size is explicitly given via adapter params
+  var paramSize = getVideoBidParam(bid, 'size');
+
+  var firstSize = [];
+  var sizes = [];
+
+  if (typeof paramSize !== 'undefined' && paramSize != '') {
+    sizes = parseSizes(paramSize);
+  } else {
+    sizes = getVideoSizes(bid);
+  }
+  firstSize = getFirstSize(sizes);
```

Comment:
Change this to `const firstSize = getFirstSize(sizes);` and remove line 217

Reply:
Hi @msm0504 updated as requested.